src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -2845,7 +2852,9 @@ namespace ReportSizingManager {
                     } else {                                                   // coil is in main air loop
                         if (PrimaryAirSystem(CurSysNum).NumOACoolCoils == 0) { // there is no precooling of the OA stream
                             AutosizeDes = FinalSysSizing(CurSysNum).MixTempAtCoolPeak;
-                        } else { // thereis precooling of the OA stream
+                        } else if (DataDesInletAirTemp > 0.0) {
```

Comment:
Is an air temperature compared to zero really correct here?  The flow greater than zero check, sure, but seems random to compare a non-absolute temperature to zero.

Reply:
These need a more defensible initialization.